optimo/models.py

```diff
@@ -324,10 +324,20 @@ class Driver(BaseModel):
     :param work_shifts: ``list`` of :class:`optimo.models.WorkShift` objects
     :param skills: ``list`` of string skill ids for the driver
     :param speed_factor: ``numbers.Number`` denoting the driving speed adjustment
+    :param service_regions: ``list`` of
+        :class:`optimo.models.ServiceRegionPolygon` denoting the areas the driver
```

Comment:
Nitpick: leave a space after `:class:`

Reply:
That's sphinx syntax. It won't render correctly otherwise.